rules/generate-insightful-reports-via-company-emails/rule.md

```diff
@@ -0,0 +1,24 @@
+---
+type: rule
+title: Do you generate insightful reports via company emails?
+uri: generate-insightful-reports-via-company-emails
```

Comment:
Please also add this rule to a category. Checkout: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Add-and-Edit-Categories-and-Top-Categories